### ISSUE TITLE
Missing git in the docker image

### ISSUE THREAD

**Issue by Lorca-R** [jenkinsci/docker-ssh-agent#164]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

not related

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

ubuntu 18.04

### Reproduction steps

git-lfs is missing in the latest image (Oct 8, 2022 at 6:19 pm) push 

[docker hub jenkins/ssh-agent:latest](https://hub.docker.com/layers/jenkins/ssh-agent/latest-jdk11/images/sha256-ad03daf40b60d05c4e84d03e8d5b65b4c5b797e2beea134545f82de2b4070d0b?context=explore)
in layer 10 shows following command: 
```
RUN |5 user=jenkins group=jenkins uid=1000 gid=1000 JENKINS_AGENT_HOME=/home/jenkins /bin/sh -c apt-get update     && apt-get install --no-install-recommends -y openssh-server     && rm -rf /var/lib/apt/lists/* # buildkit
```

but in master  https://github.com/jenkinsci/docker-ssh-agent/blob/c8cb1e99a0ae3c0aed14bc7fbbe6d0cc0996064b/8/bullseye/Dockerfile#L39
```
RUN apt-get update \
    && apt-get install --no-install-recommends -y openssh-server git-lfs netcat-traditional \
    && rm -rf /var/lib/apt/lists/*
```
`git-lfs netcat-traditional` is missing. Clearly there is a mismatch between git repo and actual docker image.

I checked some other images like `jenkins/ssh-agent:jdk11` and `jenkins/ssh-agent:alpine`, git-lfs is not installed too.

Is it removed for some reason? 

### Expected Results

git is correctly installed

### Actual Results

git is not installed

**Comment by dduportal** [comment]
Hi @Lorca-R sorry for coming back so late at you.

It looks like it is fixed with the recent release 4.5.0. Causes was the same as https://github.com/jenkinsci/docker-ssh-agent/issues/174: older images were overriding the `latest*` tags.


```console
$ docker pull jenkins/ssh-agent:latest
latest: Pulling from jenkins/ssh-agent
a8ca11554fce: Already exists 
741e5c1b0ffe: Pull complete 
13f147abde95: Pull complete 
bc5b4ac187e0: Pull complete 
4f4fb700ef54: Pull complete 
4b126f4f4832: Pull complete 
fd4fd8ca96f4: Pull complete 
4382a4dc556c: Pull complete 
Digest: sha256:63105d6130e3bf87c1867842da45481c068bef1d297709610dee11cc554a671a
Status: Downloaded newer image for jenkins/ssh-agent:latest
$ docker.io/jenkins/ssh-agent:latest
docker run --rm --entrypoint='' jenkins/ssh-agent:latest dpkg -l | grep git 
ii  dash                      0.5.11+git20200708+dd9ef66-5   amd64        POSIX-compliant shell
ii  git                       1:2.30.2-1                     amd64        fast, scalable, distributed revision control system
ii  git-lfs                   2.13.2-1+b5                    amd64        Git Large File Support
ii  git-man                   1:2.30.2-1                     all          fast, scalable, distributed revision control system (manual pages)
ii  librtmp1:amd64            2.4+20151223.gitfa8646d.1-2+b2 amd64        toolkit for RTMP streams (shared library)
```

Can you confirm?

**Comment by dduportal** [comment]
Closing as it should be fixed. Please feel free to reopen if it was a mistake.
